modules/setting/setting.go

```diff
@@ -867,6 +867,16 @@ func newLogService() {
 	LogModes = strings.Split(Cfg.Section("log").Key("MODE").MustString("console"), ",")
 	LogConfigs = make([]string, len(LogModes))
 
+	useConsole := false;
+	for _, mode := range LogModes {
+		if "console" == mode {
```

Comment:
is this "safer" or just a code style ? in Java (yes that is why there are ; every where 😄 ) the "console".equals(mode) is null safe  

still followed your proposal